modules/ding_debt_easy/lib/Nets/Easy/Checkout.inc

```diff
@@ -0,0 +1,208 @@
+
+  /**
+   * Default constructor.
+   */
+  public function __construct() {
+    $this->order = new Order();
+  }
+
+  /**
+   * Get integration type.
+   *
+   * @return string
+   *   The integration type (HostedPaymentPage or EmbeddedCheckout).
+   */
+  public function getIntegrationType() {
+    return $this->integrationType;
+  }
+
+  /**
+   * Set integration type.
+   *
+   * @param string $integrationType
+   *   The integration type (HostedPaymentPage or EmbeddedCheckout).
+   *
+   * @return $this
+   */
+  public function setIntegrationType($integrationType) {
+    $this->integrationType = $integrationType;
+
+    return $this;
+  }
+
+  /**
+   * Get terms declaration URL.
+   *
+   * @return string
+   *   The URL to terms declaration.
+   */
+  public function getTermsUrl() {
+    return $this->termsUrl;
+  }
+
+  /**
+   * Set terms declaration URL.
+   *
+   * @param string $termsUrl
+   *   The URL to use for terms declaration.
+   *
+   * @return $this
+   */
+  public function setTermsUrl($termsUrl) {
+    $this->termsUrl = $termsUrl;
+
+    return $this;
+  }
+
+  /**
+   * Get privacy declaration URL.
+   *
+   * @return string
+   *   The URL to privacy declaration.
+   */
+  public function getMerchantTermsUrl() {
+    return $this->merchantTermsUrl;
+  }
+
+  /**
+   * Get privacy declaration URL.
+   *
+   * @param string $merchantTermsUrl
+   *   The URL to privacy declaration.
+   *
+   * @return $this
+   */
+  public function setMerchantTermsUrl($merchantTermsUrl) {
+    $this->merchantTermsUrl = $merchantTermsUrl;
+
+    return $this;
+  }
+
+  /**
+   * Get return URL.
+   *
+   * @return string
+   *   The return URL to use.
+   */
+  public function getReturnUrl() {
+    return $this->returnUrl;
+  }
+
+  /**
+   * Set return URL.
+   *
+   * @param string $returnUrl
+   *   The URL to set as return URL.
+   *
+   * @return $this
+   */
+  public function setReturnUrl($returnUrl) {
+    $this->returnUrl = $returnUrl;
+
+    return $this;
+  }
+
+  /**
+   * Get cancel URL.
+   *
+   * @return string
+   *   The cancel URL to use.
+   */
+  public function getCancelUrl() {
+    return $this->cancelUrl;
+  }
+
+  /**
+   * Set cancel URL.
+   *
+   * @param string $cancelUrl
+   *   The URL to set as cancel URL.
+   *
+   * @return $this
+   */
+  public function setCancelUrl($cancelUrl) {
+    $this->cancelUrl = $cancelUrl;
+
+    return $this;
+  }
+
+  /**
+   * Get order.
+   *
+   * @return \Nets\Easy\Order
+   *   The order for this checkout.
+   */
+  public function getOrder() {
+    return $this->order;
+  }
+
+  /**
+   * Set order.
+   *
+   * @param string $order
+   *   Set order to the checkout.
+   *
+   * @return $this
+   */
+  public function setOrder($order) {
+    $this->order = $order;
+
+    return $this;
+  }
+
+  /**
+   * Build and encoded the checkout as json string.
+   *
+   * @return false|string
+   *   JSON string or FALSE on error.
+   */
+  public function toJson() {
```

Comment:
I think it would be good to have all your models have to `toArray()` function and let JSON encoding be a Client decision.

Reply:
Fixed